src/axom/multimat/multimat.cpp

```diff
@@ -649,20 +674,12 @@ void MultiMat::makeOtherRelation()
     }
   }
 
-  newRel = new StaticVariableRelationType(&set2, &set1);
-  newRel->bindBeginOffsets(set2.size(), &newBeginVec);
-  newRel->bindIndices(newIndicesVec.size(), &newIndicesVec);
+  newRel = StaticVariableRelationType(&set2, &set1);
+  newRel.bindBeginOffsets(set2.size(), &newBeginVec);
+  newRel.bindIndices(newIndicesVec.size(), &newIndicesVec);
 
-  if(newRelptr == &m_matCellRel)
-  {
-    m_matCellNZSet = new RelationSetType(newRel);
-    m_matCellProdSet = new ProductSetType(&set2, &set1);
-  }
-  else
-  {
-    m_cellMatNZSet = new RelationSetType(newRel);
-    m_cellMatProdSet = new ProductSetType(&set2, &set1);
-  }
+  getRelSparseSet(layout) = RelationSetType(&newRel);
```

Comment:
Suggestion (minor) -- since `getRelSparseSet()` isn't just an accessor, but can also be a mutator, would it make sense to just name the function `relSparseSet()` (i.e. without the `get`)